server/core/src/main/java/org/infinispan/server/core/configuration/SslEngineConfigurationBuilder.java

```diff
@@ -125,7 +136,7 @@ public void validate() {
       if(domain == null) {
          throw log.noSniDomainConfigured();
       }
-      if (sslContext == null) {
+      if (sslContextSupplier == null) {
```

Comment:
missing checking the supplier
```suggestion
      if (sslContextSupplier == null || sslContextSupplier.get() == null) {
```

Reply:
Fixed